modules/aggregation-layers/src/heatmap-layer/heatmap-layer.ts

```diff
@@ -44,23 +44,24 @@ import {
   log,
   Position,
   UpdateParameters,
-  DefaultProps
+  DefaultProps,
+  project32
 } from '@deck.gl/core';
 import TriangleLayer from './triangle-layer';
 import AggregationLayer, {AggregationLayerProps} from '../aggregation-layer';
 import {defaultColorRange, colorRangeToFlatArray} from '../utils/color-utils';
 import weightsVs from './weights-vs.glsl';
 import weightsFs from './weights-fs.glsl';
-import vsMax from './max-vs.glsl';
-import fsMax from './max-fs.glsl';
+import maxVs from './max-vs.glsl';
+import maxFs from './max-fs.glsl';
 
 const RESOLUTION = 2; // (number of common space pixels) / (number texels)
 const TEXTURE_PROPS: TextureProps = {
   format: 'rgba8unorm',
   mipmaps: false,
   sampler: {
-    minFilter: 'nearest',
-    magFilter: 'nearest',
+    minFilter: 'linear',
```

Comment:
In v8, these are also LINEAR. This [PR introduced the regression](https://github.com/visgl/deck.gl/commit/1c4f9a99596b3f913426f7d5df3d7e831b4e99c0#diff-d67a7dcf9ad34b7dd75dfec7c581805a329bf62908a163b9eef33d7f872b08c8L66-L72)